lib/lua/lmathlib.c

```diff
@@ -207,6 +208,7 @@ static int math_random (lua_State *L) {
 
 
 static int math_randomseed (lua_State *L) {
+    // DISCUSSME: what do we do here?
   srand(luaL_checkint(L, 1));
```

Comment:
Probably the same thing that sexps that manipulate the RNG seed do.

Reply:
As per @notimaginative 's comments, I'll be removing all changes I made in the `lib` subfolder.